src/extractors/custom/deadline.com/index.js

```diff
@@ -0,0 +1,34 @@
+export const DeadlineComExtractor = {
+  domain: 'deadline.com',
+
+  title: {
+    selectors: ['title'],
+  },
+
+  author: {
+    selectors: ['div.author__byline p a span:first-of-type'],
```

Comment:
This selector could be simplified/rewritten, it's a bit nested and "fragile". I've noticed there are other simpler selectors to match the author name from elements on the page. Could you please look into that?

Reply:
That's great, you're welcome! 😄